src/vars.js

```diff
@@ -25,36 +25,32 @@ exports.ecmaIdentifiers = {
   isFinite           : false,
   isNaN              : false,
   JSON               : false,
-  Map                : false,
   Math               : false,
   Number             : false,
   Object             : false,
-  Proxy              : false,
-  Promise            : false,
   parseInt           : false,
   parseFloat         : false,
   RangeError         : false,
   ReferenceError     : false,
   RegExp             : false,
-  Set                : false,
   String             : false,
   SyntaxError        : false,
   TypeError          : false,
-  URIError           : false,
-  WeakMap            : false,
-  WeakSet            : false
+  URIError           : false
 };
 
-exports.newEcmaIdentifiers = {
-  Set     : false,
-  Map     : false,
-  WeakMap : false,
-  WeakSet : false,
-  Proxy   : false,
-  Promise : false,
-  Reflect : false,
-  Symbol  : false,
-  System  : false
+exports.esVersion = {
+  6: {
+    Set     : false,
+    Map     : false,
+    WeakMap : false,
+    WeakSet : false,
+    Proxy   : false,
+    Promise : false,
+    Reflect : false,
+    Symbol  : false,
+    System  : false
+  }
```

Comment:
does the System object still exist? It was removed from the draft, and I thought it was going to be defined in a separate document but I haven't seen it anywhere if that happened